docs/send-data/installed-collectors/index.md

```diff
@@ -1,193 +1,50 @@
-
-Unlike the fixed size cache, which evicts old data to make room for new data, flushing mode stops collection of new data and focuses only on sending existing data (flushing the cache).
-
-A Collector enters flushing mode when less than 10% of free disk space remains on the disk where the Collector is installed. For more information, see [Flushing Mode](/docs/send-data/collector-faq#flushing-mode).
-
-## Collector monitoring and logging
-
-An Installed Collector sends a heartbeat to the Sumo service every 15 seconds. If the Sumo service does not receive a heartbeat for 30 minutes, it considers the Collector to be offline, and shows its health status as red in the **Collection** page of the Sumo web app. The heartbeat is linked to the [`alive` parameter](/docs/send-data/use-json-configure-sources)in the JSON object. If an Installed Collector
-appears offline try restarting the service and [testing connectivity](collector-installation-reference/test-connectivity-sumo-collectors.md).
-
-The Collector uses the log4j2 framework. You can tailor log rotation behavior for `collector.log` by editing the `log4j2.xml` file in the collector’s `/config` directory. For more information, see Log Rotation Settings.
+Our Installed Collector is installed on your local environment, as opposed to our Hosted Collector, which resides on the Sumo Logic service. After installing a Collector, you'll need to add Sources, to which the Collector connects to obtain data to send to the Sumo Logic service. 
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In this section, we'll introduce the following concepts:
+
+<div className="box-wrapper" markdown="1">
+<div className="box smallbox1 card">
+  <div className="container">
+  <a href="/docs/send-data/installed-collectors/configuration"><img src={useBaseUrl('img/icons/operations/data-collection.png')} alt="icon" width="30"/><h4>Install and Configure a Collector</h4></a>
+  <p>Install and configure a collector to gather and send data to Sumo Logic.</p>
+  </div>
+</div>
+<div className="box smallbox2 card">
+  <div className="container">
+  <a href="/docs/send-data/installed-collectors/macos"><img src={useBaseUrl('img/icons/operations/data-collection.png')} alt="icon" width="30"/><h4>For macOS</h4></a>
+  <p>Install or uninstall a Collector on a macOS system.</p>
+  </div>
+</div>
+<div className="box smallbox3 card">
+  <div className="container">
+  <a href="/docs/send-data/installed-collectors/linux"><img src={useBaseUrl('img/icons/operations/data-collection.png')} alt="icon" width="30"/><h4>For Linux</h4></a>
+  <p>Install or uninstall a Collector on a Linux system.</p>
+  </div>
+</div>
+  <div className="box smallbox4 card">
+  <div className="container">
+  <a href="/docs/send-data/installed-collectors/windows"><img src={useBaseUrl('img/icons/operations/data-collection.png')} alt="icon" width="30"/><h4>For Windows</h4></a>
+  <p>Install or uninstall a Collector on a Windows system.</p>
+  </div>
+</div>
```

Comment:
One last thing, @kimsauce. Windows should be card 3, and Linux should be card 4.

Reply:
I ended up switching the order in sidebars (mac > linux > windows order preferable)